src/components/Demo/index.js

```diff
@@ -66,6 +76,10 @@ export const DemoComponent = () => {
           <textarea onChange={e => onDeckChange(e)} defaultValue={0} />
           <JetsButton className="jets-btn jets-demo--btn" content="SET DECK" onClick={onSetDeck} />
         </div>
+        <div className="jets-demo--controller">
+          <textarea onChange={e => onSetDefaultSeatJumpTo(e)} defaultValue={defaultSeatJumpTo} />
+          <JetsButton className="jets-btn jets-demo--btn" content="SEAT JUMP TO" onClick={onSetSeatJumpTo} />
```

Comment:
add `reset` to the demo, because every next sent prop will also invoke `seatJumpTo`